frontend/src/pages/product-overview/ProductOverview.jsx

```diff
@@ -58,6 +94,20 @@ function ProductOverview() {
                 Starts from&nbsp;
                 <span className="purple-span">${product.startingPrice}</span>
               </p>
+              { product.user.id !== user.id &&
+                <div className="enter-bid-container">
+                  <Input
+                    value={enteredPrice}
+                    onChange={(e) => setPrice(e.target.value)}
+                    width="111px"
+                    type="gray"
+                  ></Input>
+                  <Button onClick={placeBid}>
+                    PLACE BID <FontAwesomeIcon icon={faAngleRight} />{" "}
+                  </Button>
+                  <p>Enter ${productBidInfo ? productBidInfo.highestBid : product.startingPrice } or more</p>
```

Comment:
But if there are bids, he cannot enter the `highestBid`, he must enter a larger bid, this message is misleading.